client/src/components/InputHelperText.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Anchor, Markdown } from 'grommet'
+import getHelperText from 'helpers/getHelperText'
+import styled from 'styled-components'
+
+const StyledSpan = styled.span`
+  font-style: italic;
+  font-size: 12px;
+  color: #666666;
+`
+
+// Override default anchor behavior to open link in new tab
+const ExternalAnchor = (props) => {
+  /* eslint-disable-next-line */
+  return <Anchor {...props} target="_blank"/>
+}
+
+// override links to open in new tab
```

Comment:
Was the a note to yourself that the above block addressed?

Reply:
yeah that comment can go